.lintrunner.toml

```diff
@@ -854,6 +854,8 @@ include_patterns = [
     'test/distributed/_composable/**/*.py',
     'torch/testing/_internal/common_dist_composable.py',
     'test/test_value_ranges.py',
+    'torch/utils/_sympy/interp.py',
```

Comment:
Isn't this a bit extreme just for something that could have been a type ignore comment?

Reply:
this is the UFMT section lol

Reply:
Oh lol.